src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+  height: 16px;
+
+  &__item {
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+    &:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+      background-repeat: no-repeat;
+      background-position: center;
+    }
```

Comment:
Remember that each product card should have stars with `stars--N` modifier. Same as it was in product card and stars layout tasks